website/content/en/docs/building-operators/golang/tutorial.md

```diff
@@ -212,10 +200,66 @@ There are a number of other useful configurations that can be made when initialz
 - Filter watch events using [predicates][event_filtering]
 - Choose the type of [EventHandler][event_handler_godocs] to change how a watch event will translate to reconcile requests for the reconcile loop. For operator relationships that are more complex than primary and secondary resources, the [`EnqueueRequestsFromMapFunc`][enqueue_requests_from_map_func] handler can be used to transform a watch event into an arbitrary set of reconcile requests.
 
+### Reconcile loop
+
+The reconcile function is responsible for synchronizing the desired state as represented by their Specs and the actual state of the system. In this way, it works like a loop, and it does not stop until all conditionals match its implementation. The following is pseudo-code with an example that clarifies it.
+
+```go
+reconcile App {
+ 
+   // Check if a Deployment for the app exists, if not create one
+   // If has an error, then go to the beginning of the reconcile
+   if err != nil {
+       return reconcile.Result{}, err 
+   } 
+    
+   // Check if a Service for the app exists, if not create one 
+   // If has an error, then go to the beginning of the reconcile
+   if err != nil {
+       return reconcile.Result{}, err 
+   }  
+ 
+   // Looking for Database CR/CRD 
+   // Check the Database Deployments Replicas size
+   // If deployment.replicas size != cr.size, then update it
+   // Then, go to the beginning of the reconcile
+   if err != nil {
+       return reconcile.Result{Requeue: true}, nil
+   }  
+   ...
+    
+   // If it is at the end of the loop, then:
+   // All was done successfully and the reconcile can stop  
+   return reconcile.Result{}, nil
+ 
+}
+```
 
-#### Reconcile loop
+The following are a few possible return options to restart the Reconcile:
 
-Every Controller has a Reconciler object with a `Reconcile()` method that implements the reconcile loop. The reconcile loop is passed the [`Request`][request-go-doc] argument which is a Namespace/Name key used to lookup the primary resource object, Memcached, from the cache:
+- With the error:
+```go
+return ctrl.Result{}, err
+```
+
+- Without an error:
+```go
+return ctrl.Result{Requeue: true}, nil
+```
+
+- Therefore, to stop the Reconcile, use:
+```go
+return ctrl.Result{}, nil
+```
+
+- Reconcile again after X time:
+```go
+return ctrl.Result{RequeueAfter: nextRun.Sub(r.Now())}, nil
+```
```

Comment:
Already discussed above in the Reconcile loop, so this is redundant 
```suggestion
```

Reply:
we did not describe the result options. See the lines 236 to 235. We need to keep it or these lines. what do you prefer?